packages/angular/projects/ui-angular/src/lib/primitives-poc/view/view-component/ComponentSetting.ts

```diff
@@ -0,0 +1,16 @@
+export class ComponentSettings {
+  /** Component selector String */
+  public static COMPONENTNAME = '[amplify-view]';
```

Comment:
Nit: prefer `COMPONENT_SELECTOR` to be explicit

Reply:
Done.